pkg/investigations/ccam/ccam.go

```diff
@@ -93,6 +99,9 @@ func RemoveLimitedSupport(cluster *v1.Cluster, ocmClient ocm.Client, pdClient pa
 		logging.Info("Primary has been alerted")
 		return nil
 	}
+	if removedReason {
+		metrics.LimitedSupportLifted.WithLabelValues(r.AlertType.String(), r.PdClient.GetEventType(), ccamLimitedSupport.Summary).Inc()
+	}
```

Comment:
**thought:** While I think it's great to decouple this from the logic of deleting the limited support reason, I'm wondering how this will evolve in the future as this currently allows forgetting to create/update a metric. 

A wrapper around the function to also increment metrics also seems wrong-ish. Not sure what the correct way to implement this is, but I'm totally fine leaving it decoupled.